src/backend/distributed/executor/multi_executor.c

```diff
@@ -66,7 +66,8 @@ static CustomExecMethods RouterSingleModifyCustomExecMethods = {
 	.ExplainCustomScan = CitusExplainScan
 };
 
-static CustomExecMethods RouterMultiModifyCustomExecMethods = {
+/* not static to enable swapping in multi-modify logic during router execution */
+CustomExecMethods RouterMultiModifyCustomExecMethods = {
```

Comment:
We're not doing anything about multi-shard update/delete for the moment, but I guess there could be a future in which multiple functions differ. Fine leaving this as is.